data/categories/ae_arts_entertainment.yml

```diff
@@ -7740,6 +7743,201 @@
   - instrument_bow_material
   - instrument_material
   - string_instrument_size
+- id: ae-2-8-7-7
+  name: Banjos
+  children:
+  - ae-2-8-7-7-1
+  - ae-2-8-7-7-2
+  - ae-2-8-7-7-3
+  - ae-2-8-7-7-4
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-1
+  name: 4-String Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-2
+  name: 5-String Banjoss
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-3
+  name: 6-String Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-4
+  name: Tenor Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-8
+  name: Mandolins
+  children:
+  - ae-2-8-7-8-1
+  - ae-2-8-7-8-2
+  - ae-2-8-7-8-3
+  - ae-2-8-7-8-4
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-8-1
+  name: A-Style Mandolins
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-8-2
+  name: F-Style Mandolins
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-8-3
+  name: Bowlback Mandolins
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
```

Comment:
Since bowl-back refers to the back shape and both A-style and F-style mandolins could have that back shape, I would skip `Bowlback Mandolins` as a category and instead add an attribute for `Mandolin back shape`, including values like Bowl-back, Flat-back and Archtop -- to avoid overlaps.
![image](https://github.com/user-attachments/assets/7404bd0a-0db0-404c-9dcb-b82f35c8d0b5)